Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -197,9 +197,10 @@ public static IEnumerable<object[]> EnumTypes
         {
             get
             {
-                return typeof(IGitHubClient).GetTypeInfo().Assembly.ExportedTypes
-                    .Where(type => type.GetTypeInfo().IsEnum)
-                    .Where(type => !ExcludedEnumTypes.Contains(type))
+                return GetModelTypes(includeRequestModels: true)
+                    .SelectMany(type => type.GetProperties())
+                    .SelectMany(property => UnwrapGenericArguments(property.PropertyType))
+                    .Where(type => type.Namespace.StartsWith("Octokit") && type.GetTypeInfo().IsEnum)
```

Comment:
minor nitpick but existing convention tests match the assembly to that of `IGitHubClient` rather than hardcode "Octokit".  I guess we'd never change assemblies but I feel like it's cleaner to not hardcode it... ?